src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildASelfContainedApp.cs

```diff
@@ -374,6 +374,35 @@ static int Last2DigitsTo0(int versionBuild)
             publishCommand.Execute(new [] {"-property:SelfContained=true", "-property:_CommandLineDefinedSelfContained=true", $"-property:RuntimeIdentifier={rid}", "-property:_CommandLineDefinedRuntimeIdentifier=true" }).Should().Pass().And.NotHaveStdOutContaining("warning");
         }
 
+        [Theory]
+        [InlineData("net7.0")]
```

Comment:
Do we want a Theory with only one instance?  Should we have a fact that just uses `ToolsetInfo.CurrentTargetFramework` instead?

Reply:
Good point